lib/zipkin-tracer/faraday/zipkin-tracer.rb

```diff
@@ -50,14 +45,14 @@ def trace!(env, trace_id)
       remote_endpoint = Trace::Endpoint.remote_endpoint(url, @service_name, local_endpoint.ip_format) # The endpoint we are calling.
       @tracer.with_new_span(trace_id, env[:method].to_s.downcase) do |span|
         # annotate with method (GET/POST/etc.) and uri path
-        span.record(Trace::BinaryAnnotation.new(URI_KEY, url.path, STRING_TYPE, local_endpoint))
-        span.record(Trace::BinaryAnnotation.new(SERVER_ADDRESS, SERVER_ADDRESS_SPECIAL_VALUE, BOOLEAN_TYPE, remote_endpoint))
-        span.record(Trace::Annotation.new(Trace::Annotation::CLIENT_SEND, local_endpoint))
+        span.record_tag(Trace::BinaryAnnotation::URI, url.path, Trace::BinaryAnnotation::Type::STRING, local_endpoint)
```

Comment:
I'm merging this for now, eventually I feel things will be moved around, good enough for now